File: Resources/Projects/Archive/2023/Competition/Programming/Contest/TOI-Zero_68/A2/019/TOI-Zero_68/A1/023/023.py
a = int(input())
b = input().strip().upper()
c = float(a)
if b == 'F':
    c = (a - 32) * 5.0 / 9.0

if c <= 0:
    print("solid")
elif c < 100:
    print("liquid")
else:
    print("gas")

File: Resources/Projects/Archive/2023/Competition/Programming/Contest/TOI-Zero_68/A2/019/TOI-Zero_68/A1/023/023.c
#include <stdio.h>

int main() {
    int a;
    char b;
    scanf("%d", &a);
    scanf(" %c", &b);
    
    float c = a;
    if (b == 'F') {
        c = (a - 32) * 5.0 / 9.0;
    }
    
    if (c <= 0) {
        printf("solid\n");
    } else if (c < 100) {
        printf("liquid\n");
    } else {
        printf("gas\n");
    }
    
    return 0;
} 